src/main/webapp/app/localci/build-agents/build-agent-summary/build-agent-summary.component.ts

```diff
@@ -60,7 +58,6 @@ export class BuildAgentsComponent implements OnInit, OnDestroy {
 
     private updateBuildAgents(buildAgents: BuildAgent[]) {
         this.buildAgents = buildAgents;
-        this.setRecentBuildJobsDuration(buildAgents);
```

Comment:
Do I understand it properly that this information is now provided by `BuildAgentsService .getBuildAgentDetails`?

Reply:
Yes the build agent details page now shows the duration of the recent build jobs